components/src/core/ChannelValue/ChannelValue.test.tsx

```diff
@@ -1,49 +1,56 @@
 import React from 'react';
-import { createRoot } from 'react-dom/client';
-import { findByTestId, mountWithTheme } from '../test-utils';
+import { render, screen, cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom';
 import { ChannelValue } from './ChannelValue';
-import * as Enzyme from 'enzyme';
-import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
 import Menu from '@mui/icons-material/Menu';
 import { createTheme, ThemeProvider } from '@mui/material/styles';
 import * as BLUIThemes from '@brightlayer-ui/react-themes';
 
 const theme = createTheme(BLUIThemes.blue);
 
-Enzyme.configure({ adapter: new Adapter() });
+afterEach(cleanup);
 
 describe('ChannelValue', () => {
     it('renders without crashing', () => {
-        const div = document.createElement('div');
-        const root = createRoot(div);
-        root.render(
+        render(
             <ThemeProvider theme={theme}>
                 <ChannelValue value={'test'} />
             </ThemeProvider>
         );
+        expect(screen.getByText('test')).toBeTruthy();
     });
     it('should render with the wrapper class', () => {
-        const wrapper = mountWithTheme(<ChannelValue value={1} />, theme);
-        expect(findByTestId('wrapper', wrapper)).toBeTruthy();
-        wrapper.unmount();
+        render(
+            <ThemeProvider theme={theme}>
```

Comment:
What is the difference between this test vs the one above it, "renders without crashing"?

Reply:
removed dup test